<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecorderNewDialog</class>
 <widget class="QDialog" name="RecorderNewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New recording</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources/icons/icons.qrc">
    <normaloff>:/icons/actions/16/media-record.png</normaloff>:/icons/actions/16/media-record.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="box">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0" rowspan="2">
       <widget class="QLabel" name="labelSelected">
        <property name="text">
         <string>Selected channel:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelName">
        <property name="text">
         <string>Recording name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="editName"/>
      </item>
      <item row="1" column="1" rowspan="2" colspan="2">
       <widget class="QLabel" name="valueSelected">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ChannelsDisplayWidget" name="channelsWidget" native="true"/>
   </item>
   <item>
    <widget class="FancyLineEdit" name="search" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttons">
     <item>
      <spacer name="spacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRecord">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Record</string>
       </property>
       <property name="icon">
        <iconset resource="../../../resources/icons/icons.qrc">
         <normaloff>:/icons/actions/16/media-record.png</normaloff>:/icons/actions/16/media-record.png</iconset>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonTimer">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Add new timer</string>
       </property>
       <property name="icon">
        <iconset resource="../../../resources/icons/icons.qrc">
         <normaloff>:/icons/actions/16/list-add.png</normaloff>:/icons/actions/16/list-add.png</iconset>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../../../resources/icons/icons.qrc">
         <normaloff>:/icons/actions/16/window-close.png</normaloff>:/icons/actions/16/window-close.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChannelsDisplayWidget</class>
   <extends>QWidget</extends>
   <header>channels/widgets/ChannelsDisplayWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FancyLineEdit</class>
   <extends>QWidget</extends>
   <header>style/FancyLineEdit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>buttonTimer</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../../../resources/icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
